stix2/__init__.py

```diff
@@ -19,27 +19,11 @@
 
 # flake8: noqa
 
-from . import exceptions
-from .common import (TLP_AMBER, TLP_GREEN, TLP_RED, TLP_WHITE, CustomMarking,
-                     ExternalReference, GranularMarking, KillChainPhase,
-                     MarkingDefinition, StatementMarking, TLPMarking)
-from .core import Bundle, _register_type, parse
+from . import exceptions, v20
```

Comment:
I'm not sure this line is needed. `stix2.exceptions` and `stix2.v20` should both be importable even without this line.

Reply:
You are correct. We can remove these.